test/user.js

```diff
@@ -18,6 +18,28 @@ const api = require('./lib/api.js');
 const prompt = require('../lib/prompt.js');
 const user = require('../lib/user.js');
 const util = require('../lib/util.js');
+const constants = require('./../lib/constants');
+
+const invalidEmail = 'invalid@example.com';
+const invalidPassword = 'invalidPass';
+
+const helper = {
```

Comment:
I'm not sure that these functions need to be nested inside of an object (since it's not being exported anywhere) but if you choose to leave it this way, I find that it's best to be as descriptive as possible with variable naming - e.g. `loginTestsHelper` - especially since this is public-facing code